### PR TITLE
Remove examples.build/. Let gulp server handle the lib location substitution dynamically at serving time.

### PR DESCRIPTION
**Pull request by lannka**
#4221

### CODE REVIEW

**Comment by cramforce** [comment]
This is great. Will need to re-tune my muscle memory :)

File: gulpfile.js [hunk 1]
```diff
@@ -340,112 +338,6 @@ function checkTypes() {
 }
 
 /**
- * Build the examples
```
Comment (erwinmombay) [inline gulpfile.js]: awesome

File: build-system/server.js [hunk 2]
```diff
@@ -302,11 +287,39 @@ app.use('/min/', function(req, res) {
   proxyToAmpProxy(req, res, /* minify */ true);
 });
 
-app.use('/examples.build/analytics.config.json', function (req, res, next) {
+app.use('/examples/analytics.config.json', function(req, res, next) {
   res.setHeader('AMP-Access-Control-Allow-Source-Origin', getUrlPrefix(req));
   next();
 });
 
+app.get('/examples/*', function(req, res, next) {
```
Comment (lannka) [inline build-system/server.js]: ha, that was my first try. it worked great, except for those files under a sub directory, such as /examples/pwa/pwa.html

**Comment by erwinmombay** [comment]
seeing this error, not sure if expected
screen shot 2016-07-28 at 12 50 17 pm

**Comment by lannka** [comment]
Good catch. Fixed.  
@erwinmombay Since you've already have it compiled locally, could you help check the examples of live-list and live-blog?

**Comment by erwinmombay** [comment]
@lannka sure

**Comment by erwinmombay** [comment]
@lannka all the localhost hosted images are currently broken, see http://localhost:8000/examples/article.amp.html/max

**Comment by erwinmombay** [comment]
LGTM once you fix the image loading issue

**Comment by lannka** [comment]
Fixed all image loading. PTAL.

**Comment by lannka** [comment]
@erwinmombay as discussed, reverted back to old URL format: `abc.amp.(max|min).html` 
PTAL

**Comment by erwinmombay** [comment]
just need the change to `/examples/live-blog(-non-floating-button)?.amp.html` to support `max/min`. otherwise LGTM!

**Pull request merged**